### ISSUE TITLE
Missing label on “follow logs” button

### ISSUE THREAD

**Issue by jgwhite** [hashicorp/waypoint#3178]
**Describe the bug**
Currently the “follow logs” button does not have an accessible label.

https://github.com/hashicorp/waypoint/blob/d4324825f54833b3e477622223cc0b37ca5ae3ca/ui/app/components/render-terminal/index.hbs#L3-L5

**Expected behavior**
The button should have an accessible label. We may want to use [`<Hds::Button>`](https://design-system-components-hashicorp.vercel.app/components/button) to achieve this.

Success criteria: [WCAG 2.1 Success Criterion 1.1.1 Non-text Content](https://www.w3.org/TR/WCAG21/#non-text-content)

**Why didn’t tests catch this?**
I imagine we don’t encounter this particular state in our tests.

**Waypoint Version**
d4324825f54833b3e477622223cc0b37ca5ae3ca
